python/cog/director/director.py

```diff
@@ -313,6 +338,28 @@ def _handle_message(self, message_id: str, message_json: str) -> None:
             log.info("prediction succeeded")
             self._record_success()
 
+        self._set_span_attributes_from_tracker(span, tracker)
+
+    # OpenTelemetry is very picky about not accepting None types
+    def _set_span_attributes_from_tracker(self, span, tracker):
+        span.set_attribute("prediction.id", tracker._response.id)
+        if tracker._response.version:
+            span.set_attribute("prediction.model.version", tracker._response.version)
```

Comment:
I think we _probably_ want this to be
```suggestion
            span.set_attribute("model.version", tracker._response.version)
```